package.json

```diff
@@ -13,7 +13,10 @@
     "node": ">=6"
   },
   "bin": "index.js",
-  "files": ["index.js", "src"],
+  "files": [
```

Comment:
It looks like pre-commit hook didn’t run on this file. The formatting for these section shouldn’t have changed. Please run prettier or the precommit hook (should have been installed after the checkout and `yarn install`)